oggopusEncoder.js

```diff
@@ -78,7 +78,9 @@ OggOpusEncoder.prototype.encode = function( buffers ) {
 };
 
 OggOpusEncoder.prototype.encodeFinalFrame = function() {
-  this.encode( new Float32Array( this.encoderBufferLength - this.encoderBufferIndex ) );
+  this.encode(new Array(this.numberOfChannels).fill().map(function() {
+    return Float32Array(this.encoderBufferLength - this.encoderBufferIndex);
```

Comment:
This throws an error for me `(intermediate value).fill is not a function`.  Using Chrome ver 43